overlays/ci/delete-resources-2.yaml

```diff
@@ -0,0 +1,4 @@
+- op: remove
+  path: /spec/template/spec/containers/0/resources
```

Comment:
How is this (and the other `delete-resources`) used?

Reply:
Good catch! 

I've removed the resources all deployments now to try and make this as lean as possible.